ci/ci_acceptance.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+echo "Building Logstash artifacts"
+rake artifact:all
+
+echo "Acceptance: Installing dependencies"
+cd qa
+bundle install
+
+echo "Acceptance: Running the tests"
+bundle exec rake test:setup
+bundle exec rake test:ssh_config
+bundle exec rake test:acceptance:all
```

Comment:
let's wait on actually decide how do we aim to run this in CI environment, this script is here to empower it, so ....  I'm good with whatever path we take, but this script should help automated testing.